### PR TITLE
feature

### PR DESCRIPTION
**Pull request by ckahub**
build avatar component

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+      {
+        "type": "chrome",
+        "request": "launch",
+        "name": "Launch Chrome against localhost",
+        "file": "${workspaceRoot}/Components/Avatar/avatar.html"
+          
+      }
+  ]
+}
```
Comment (rohanmathur91) [inline .vscode/launch.json:15]: you can put this in .gitignore file.

**Review by rohanmathur91 — APPROVED** [review]
LGTM

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@import url('../../styles/utlity.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+.avatar{
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
```
Comment (devdeadviz) [inline Components/Avatar/avatar.css]: Try to use relative units instead of absolute units.

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+.md{
+    width: 40px;
+    height: 40px;
+}
+
+.lg{
+    width: 50px;
+    height: 50px;
+}
```
Comment (devdeadviz) [inline Components/Avatar/avatar.css]: same here

File: Components/Avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                <a href="#">Meta</a>
+                <span class="small-text">v1.0</span>
+            </div>
+            <ul class="right_Nav">
+                <li><a class="active" href="#">Docs</a></li>   
+            </ul>
+        </nav>
+
+
+
```
Comment (devdeadviz) [inline Components/Avatar/avatar.html:29]: remove extra spaces.

File: styles/navBar.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css2?family=Supermercado+One&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+ 
+}
```
Comment (devdeadviz) [inline styles/navBar.css:8]: You can add this in the reset stylesheet.

File: styles/utlity.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
+.doc_Container{
+    font-family: sans-serif;
+    width:100%;
+    margin: 50px auto 0;
```
Comment (devdeadviz) [inline styles/utlity.css:4]: Don't use absolute units.

File: styles/navBar.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    font-weight: 900;
+}
+
+.left-nav a{
+    color : rgb(48, 46, 46);
+    text-decoration: none;
+}
+
+.small-text{
+    color :rgb(90, 90, 90);
```
Comment (devdeadviz) [inline styles/navBar.css:44]: Use color variables instead of hard-coding them

File: styles/navBar.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+    font-weight: 800;
+    color: rgb(90, 90, 90);
+    text-decoration: none;
+    cursor: pointer;
+    font-size: .9rem;
+   
+}
+
+.right-nav a:hover{
+    color: rgb(58, 218, 247);
```
Comment (devdeadviz) [inline styles/navBar.css:72]: same here

File: styles/sideBar.css [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.sidebar{
+    font-family: 'Supermercado One', cursive;
+    flex: 0 0 240px;
+    margin-right: 1rem;
+    background-color: #fff;
```
Comment (devdeadviz) [inline styles/sideBar.css:13]: Use color variables

**Review by devdeadviz — CHANGES_REQUESTED** [review]
Dropped some required suggestion, have a look at it.

File: .vscode/launch.json (hunk 1 above)
Comment (ckahub) [reply .vscode/launch.json:15]: ok i will do so

File: Components/Avatar/avatar.css (hunk 2 above)
Comment (ckahub) [reply Components/Avatar/avatar.css]: made neccesary changes...thanx

**Pull request merged**